src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPage.tsx

```diff
@@ -85,7 +85,7 @@ const ActionDetailsPage = () => {
 
   return (
     <Layout>
-      <DefaultAction item={action} />
+      <DefaultAction actionData={action} />
```

Comment:
Renamed these in a few places so it's easier to understand what kind of data we're dealing with